command_attr/src/lib.rs

```diff
@@ -750,7 +750,7 @@ pub fn check(_attr: TokenStream, input: TokenStream) -> TokenStream {
 
     propagate_err!(create_declaration_validations(&mut fun, DeclarFor::Check));
 
-    let res = parse_quote!(serenity::framework::standard::CheckResult);
+    let res = parse_quote!(Result<(), serenity::framework::standard::Reason>);
```

Comment:
The absolute path for `Result` should be used in case a type alias is used with the same name.
```suggestion
    let res = parse_quote!(std::result::Result<(), serenity::framework::standard::Reason>);
```